### PR TITLE
chore: test change

### PR DESCRIPTION
**Pull request by sobolk**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-storage/src/provider-utils/awscloudformation/cfn-template-utils.ts [hunk 1]
```diff
@@ -14,6 +14,10 @@ export const getCloudFormationTemplatePath = (resourceName: string) => {
   );
 };
 
+/**
+ * Some irrelevant change
+ * @param resourceName foo
+ */
```
Comment (edwardfoyle) [inline packages/amplify-category-storage/src/provider-utils/awscloudformation/cfn-template-utils.ts:20]: test comment

**Comment by Amplifiyer** [comment]
Generic test comment

File: packages/amplify-category-storage/src/provider-utils/awscloudformation/s3-trigger-helpers.ts [hunk 2]
```diff
@@ -5,6 +5,12 @@ import { v4 as uuid } from 'uuid';
 import { categoryName, functionCategoryName } from '../../constants';
 import { FunctionServiceNameLambdaFunction, providerName } from './provider-constants';
 
+/**
+ * More changes
+ * @param context
+ * @param resourceName
+ * @param triggerFunctionName
```
Comment (0618) [inline packages/amplify-category-storage/src/provider-utils/awscloudformation/s3-trigger-helpers.ts:12]: This is a comment

**Review by edwardfoyle — APPROVED** [review]
for science!

**Review by sobolk — COMMENTED** [review]
hmmm

**Comment by sobolk** [comment]
generic comment

File: packages/amplify-category-storage/src/provider-utils/awscloudformation/s3-trigger-helpers.ts [hunk 3]
```diff
@@ -5,6 +5,12 @@ import { v4 as uuid } from 'uuid';
 import { categoryName, functionCategoryName } from '../../constants';
 import { FunctionServiceNameLambdaFunction, providerName } from './provider-constants';
 
+/**
+ * More changes
+ * @param context
```
Comment (kasobol-msft) [inline packages/amplify-category-storage/src/provider-utils/awscloudformation/s3-trigger-helpers.ts:10]: This comment is ambiguous.

**Review by kasobol-msft — COMMENTED** [review]
not a good pr

**Review by Amplifiyer — APPROVED** [review]
test approved

**Pull request closed without merging**